.travis.yml

```
@@ -0,0 +1,7 @@
+language: java
+
+jdk:
+  - openjdk8
+  
+before_install:
+ - chmod +x gradlew
```

I think the gradle wrapper already unix executable, the build command can be like this.
It should work if my PR is merged

```suggestion
 - ./gradlew jar
```